clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/cache/Keys.groovy

```diff
@@ -16,30 +16,15 @@
 
 package com.netflix.spinnaker.clouddriver.openstack.cache
 
+import com.netflix.frigga.Names
+import com.netflix.spinnaker.clouddriver.core.provider.agent.Namespace
 import com.netflix.spinnaker.clouddriver.openstack.OpenstackCloudProvider
 import groovy.transform.CompileStatic
 
 //TODO - Refactor to use template pattern and add to core so common code can be shared across implementations.
 @CompileStatic
 class Keys {
 
-  static enum Namespace {
-    INSTANCES
-
-    final String ns
-
-    private Namespace() {
-      def parts = name().split('_')
-
-      // Converts CONSTANT naming convention to camel case
-      ns = parts.tail().inject(new StringBuilder(parts.head().toLowerCase())) { val, next -> val.append(next.charAt(0)).append(next.substring(1).toLowerCase()) }
```

Comment:
There is one that fits our needs in the core library ... thought it would be better to use that one and adjust if needed.